CEPCPreamble.md

```diff
@@ -0,0 +1,40 @@
+# CEPC Preamble
+
+The W3C's Code of Ethics and Professional Conduct sets our to create a positive environment for W3C in which all participants feel safe and respected. It documents behaviors that are expected as well as behaviors that are unacceptable.
+It can be helpful to call attention to certain features of the code from time to time so that we are aware of what it includes. There is a lot of nuance in this code, and it is worth reading through the whole thing [link].
+
+(note to speaker - choose one or two examples each time you present) 
+
+1. Have empathy when discussing sensitive issues. You do not know about others' lived experiences. Avoid making jokes or callously mentioning sensitive topics, such as stalking, sexual assault, the pandemic, mental health, or disability. In cases when the need arises to discuss these issues and how they affect people do so with tact and empathy taking into account the gravity of the situation. 
+
+2. Be inclusive and promote diversity. Seek diverse perspectives. Diversity of views and of people powers innovation, even if it is not always comfortable. Encourage all voices. Help new perspectives be heard and listen actively. If you find yourself dominating a discussion, it is especially important to step back and encourage other voices to join in. Provide alternative ways to contribute. 
+This might mean allowing a new person to become the AC rep. This might mean paying attention to how much time you spend talking in a meeting or how often you join the queue. It might mean sharing slides a few days before a meeting so that people have an opportunity to read and translate them into their native languages. Inclusion is about making people feel welcome. 
+
+3. Think of others’ needs from their point of view. Use preferred names, titles (including pronouns), and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others and acknowledge the contributions of your peers. 
+It is impossible to be aware of all of someone else's needs without asking them what they need. This can be as simple as how to pronounce a name or more subtle, like recognizing that some people prefer to contribute by listening. Avoid making assumptions others' preferences and needs. When in doubt, ask, and respect the response. It's important to recognize the contributions of others. 

```

Comment:
Missing word: s/making assumptions others/making assumptions about others'/

Also suggest deleting this, because it was already said earlier in the same paragraph: "It's important to recognize the contributions of others."